frameworks/elastic/docs/configure.md

```diff
@@ -52,9 +55,9 @@ You can view the deploy plan for the DC/OS Elastic Service via the service URL:
 
 # Topology
 
-Each task in the cluster performs one and only one of the following roles: master, data, ingest, coordinator, kibana.
+Each task in the cluster performs one and only one of the following roles: master, data, ingest, coordinator.
 
-The default placement strategy distributes all instances of the same node task type to different agents. So no two master nodes would run on the same agent, but a data node might get placed onto an agent that is also running a master node.
+The default placement strategy has no restrictions other than distributing all the master nodes to different agents. You can specify the Marathon placement constraints for each node type. For example, you can specify that data nodes are never colocated, or that ingest nodes are deployed on a rack with high-CPU servers. 
 
```

Comment:
Suggest:
The default placement strategy specifies no constraint except that all the master nodes are distributed to different agents. You can specify further [Marathon placement constraints](http://mesosphere.github.io/marathon/docs/constraints.html) for each node type. For example....